samples/dnn/js_face_recognition.html

```diff
@@ -40,7 +40,7 @@
 
 //! [Get 128 floating points feature vector]
 function face2vec(face) {
-  var blob = cv.blobFromImage(face, 1.0 / 255, {width: 96, height: 96}, [0, 0, 0, 0], true, false)
+  var blob = cv.blobFromImage(face, 1.0, {width: 112, height: 112}, [0, 0, 0, 0], true, false)
```

Comment:
see https://github.com/opencv/opencv_zoo/blob/main/models/face_recognition_sface/demo.py#L37
https://github.com/opencv/opencv_zoo/blob/main/models/face_recognition_sface/sface.py#L15
https://github.com/opencv/opencv/blob/4.x/modules/objdetect/src/face_recognize.cpp#L45

Reply:
It makes sense to add reference to the model and parameters as comment to the code.